component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryController.java

```diff
@@ -1,79 +1,42 @@
-/*
```

Comment:
Renamed the AuditEntryController to AuditEntryControllerLegacy.  The contents of this class are the new Rest API endpoint.